Automattic/wp-calypso
# Add tracks events

Add tracks events for payment method improvements for Jetpack Manage



Closes https://github.com/Automattic/jetpack-genesis/issues/171

## Proposed Changes
Adds tracks events for updates to adding new payments in Jetpack Manage

Events include
* The `Add New Card` button is clicked
* The `Set as primary payment methods` toggle is clicked when adding a new payment
* The `Submit` button is clicked to add a new card
* The 3 dot menu below a payment card in the payment methods page is clicked
* The `Set as primary card` option in the card actions menu is clicked
* The delete option is clicked in the card actions menu

## Testing Instructions



* Use the Calypso Live link below to access Jetpack Manage > Purchases >Payment Methods page and append `?flags=jetpack/card-addition-improvements` to the url.
* Open dev tools and the Network tab. Filter all `*.gif` requests. 
* Ensure that the following events are triggered for the actions
* Clicking the Add New Card button - `calypso_partner_portal_payments_add_new_card_button_click`
![Screenshot 2024-01-16 at 7 58 18 PM](https://github.com/Automattic/wp-calypso/assets/1273880/7a8e1939-5d89-4a7d-9319-44153be53672)
* Toggle the `Set as primary payment method` checkbox in the page to add a new payment method - calypso_partner_portal_add_credit_card_as_default_toggle
![Screenshot 2024-01-16 at 7 58 52 PM](https://github.com/Automattic/wp-calypso/assets/1273880/8f15f7fe-3b9c-439e-be02-5ad4af5fbb9d)
* Clicking the Submit button to add a new payment method - `calypso_partner_portal_add_credit_card_form_submit`
![Screenshot 2024-01-16 at 7 58 52 PM](https://github.com/Automattic/wp-calypso/assets/1273880/9932e3a9-9ad6-41df-bdbf-3393b0d22a21)
* Clicking the 3 dot menu on a card in the payments method page - `calypso_partner_portal_payments_card_actions_button_click`
![Screenshot 2024-01-16 at 7 58 32 PM](https://github.com/Automattic/wp-calypso/assets/1273880/00e75595-37ce-421d-8c5b-40a42044f632)
* Clicking the Set as primary option in the card actions menu - `calypso_partner_portal_payments_card_actions_set_as_primary_click`
![Screenshot 2024-01-16 at 7 58 40 PM](https://github.com/Automattic/wp-calypso/assets/1273880/d7b88652-1b27-4c22-8e7c-0f67d234b6f2)
* Clicking the Delete option in the card actions menu - `calypso_partner_portal_payments_card_actions_delete_click`
![Screenshot 2024-01-16 at 8 09 18 PM](https://github.com/Automattic/wp-calypso/assets/1273880/f11536bc-958e-4a58-8ffe-bf33b1067926)



## Pre-merge Checklist
